### DISCUSSION TITLE
Not able to view in AR

### DISCUSSION THREAD

**Discussion by mihirm3hub** [google/model-viewer#2732] [Q&A] [answered]
I am doing some experiments with model-viewer and I am not able to see the model in AR. here is a glitch to the code: https://glitch.com/edit/#!/fiv-ar

**Comment by mihirm3hub** [comment, marked as answer]
updating Chrome fixed it, not able to sync audio properly any suggestions?

**Reply by elalish** [reply]
For some reason your Glitch link isn't working for me. I'd guess you just need to make sure all the files are loaded and ready, but not auto-playing. Then call `play()` on both MV and the audio at the same time.
